lib/game.rb

```diff
@@ -0,0 +1,64 @@
+# rubocop: disable Metrics/CyclomaticComplexity
+class Game
+  attr_reader :switch_player
+  attr_reader :who_won
+
+  WINNING_COMBINATIONS = [[1, 2, 3], [4, 5, 6], [7, 8, 9], [1, 4, 7], [2, 5, 8], [3, 6, 9], [1, 5, 9], [3, 5, 7]].freeze
```

Comment:
[OPTIONAL]
Is there a better way to get all winning combinations than hard coding it like this?

Reply:
Considering the logic of this game, using slots with an assigned number, the easier way to work is using hard code, even the number of combinations is only 8.